xesar-connect/src/main/kotlin/com/open200/xesar/connect/extension/XesarConnectComponentExt.kt

```diff
@@ -0,0 +1,100 @@
+package com.open200.xesar.connect.extension
+
+import com.open200.xesar.connect.Topics
+import com.open200.xesar.connect.XesarConnect
+import com.open200.xesar.connect.messages.command.FindComponent
+import com.open200.xesar.connect.messages.command.Query
+import com.open200.xesar.connect.messages.command.RemoteDisengage
+import com.open200.xesar.connect.messages.command.RemoteDisengagePermanent
+import com.open200.xesar.connect.messages.event.FindComponentPerformed
+import com.open200.xesar.connect.messages.event.RemoteDisengagePerformed
+import com.open200.xesar.connect.messages.event.RemoteDisengagePermanentPerformed
+import com.open200.xesar.connect.messages.query.EvvaComponent
+import com.open200.xesar.connect.messages.query.QueryList
+import java.util.*
+import kotlinx.coroutines.Deferred
+
+/**
+ * Enable/disable the beeping signal to find the component.
+ *
+ * @param installationPointId The ID of the installation point
+ * @param enable Enable or disable the beeping signal
+ * @param requestConfig The request configuration (optional).
+ */
+suspend fun XesarConnect.findComponent(
+    installationPointId: UUID,
+    enable: Boolean? = null,
+    requestConfig: XesarConnect.RequestConfig = buildRequestConfig()
+): Deferred<FindComponentPerformed> {
+    return sendCommand<FindComponent, FindComponentPerformed>(
+        Topics.Command.FIND_COMPONENT,
+        FindComponent(
+            config.uuidGenerator.generateId(), installationPointId, enable, requestConfig.token),
+        requestConfig)
+}
+
+/**
+ * Queries the list of evva components asynchronously.
+ *
+ * @param params The query parameters (optional).
+ * @param requestConfig The request configuration (optional).
+ * @return A deferred object that resolves to a response containing a list of evva components.
+ */
+suspend fun XesarConnect.queryEvvaComponentListAsync(
+    params: Query.Params? = null,
+    requestConfig: XesarConnect.RequestConfig = buildRequestConfig()
+): Deferred<QueryList.Response<EvvaComponent>> {
+    return queryListAsync(EvvaComponent.QUERY_RESOURCE, params, requestConfig)
+}
+
+/**
+ * Remotely disengage an online component.
+ *
+ * @param installationPointId The ID of the installation point
+ * @param extended false for SHORT / true for LONG disengage
+ * @param requestConfig The request configuration (optional).
+ */
+suspend fun XesarConnect.executeRemoteDisengage(
```

Comment:
should we move this to InstalltionPoint since it refers to a installationPointId.